docs/reference/2-ods-api/how-to-guides/how-to-add-api-composites-to-the-ed-fi-ods-api-solution.md

```diff
@@ -124,10 +124,7 @@ of a composite URL.
 :::note
 
 The following GitHub link contains source files for the Composite described in
-this article:
-
-* [ODS/API 7.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.2/Samples/Composites)
-* [ODS/API 7.1](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.1/Samples/Composites)
-* [ODS/API 6.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v6.2/Samples/Composites)
+this article: [ODS/API
+7.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.2/Samples/Composites)
```

Comment:
I went ahead and simplified to list only the current version here, since we are going forward with the full versioning process.

At some point we may bring in the 7.1 and 6.2 documents as well, but probably not until next year. When we do that, I'm thinking of starting from the 7.2 pages. Separately, we can use WinMerge to find the differences between the two exports from Confluence (for example, comparing the 7.2 and 7.1 exports), which will make it easy to remove anything from 7.2 that does not belong in 7.1

In other words, I don't want to go through all of this migration work again from the raw 7.1 or 6.2 exported files. Use the 7.2 markdown files as the basis and adjust them as needed.